meta-resin-common/recipes-support/resin-state-reset/resin-state-reset/resin-state-reset

```diff
@@ -9,8 +9,10 @@ populate() {
     # Re-construct directory structure and populate files
     if [ -f $2 ]; then
         mkdir -p "$(dirname $1)" && cp $2 $1
+    elif [ -d "$2" ]; then
+        mkdir -p -- "$1" && find "$2" \( -type f -o -type l \) -exec cp {} "$1" +
     else
-        mkdir -p $1 && find $2 \( -type f -o -type l \) -exec cp {} $1 +
+        mkdir -p -- "$1"
```

Comment:
/var/log/journal I suppose? Would make sense because we mount that in the state partition so it persists the logs. Do we mount /var/log/volatile? Or was that a typo?